snakes_cafe.py

```diff
@@ -276,6 +226,105 @@ def optional_menu():
 #             menu[unpacked[i][1]] = {unpacked[i][0]: [0, unpacked[i][1], unpacked[i][2]]}
 #     print(menu)
 
+
+class Order:
+    """
+    new class
+    """
+    def __init__(self):
+        self.default
+
+    def __str__(self):
+        return 'Welcome to Snakes Cafe'
+
+    def __len__(self):
+        counter = 0
+        for key, value in menu.items():
+            for k, v in value.items():
+                if v[0] != 0:
+                    counter += v[0]
+        return counter
+
+    def __repr__(self):
+        print('<Order #{} | Items: {} | Total: ${:.2f}>'.format(order_id, Order.__len__, subtotal * 1.101))
+
+    def add_item(order_line, order_quantity):
+        """
+        adds items to user order
+        """
+        global subtotal
```

Comment:
You should not be using a global `subtotal` anymore because it should be a property on the `Order` class. In fact, all the variables that are keeping track of the details of an order should be properties on `Order` rather than global variables.